Vagrantfile

```diff
@@ -9,7 +9,6 @@ Vagrant.configure("2") do |config|
   config.ssh.port = 2210
 
   config.vm.provider :virtualbox do |virtualbox|
-    virtualbox.customize ["modifyvm", :id, "--name", "ushahidi-platform"]
```

Comment:
Oops. I have to remove that for my local dev. It fails otherwise, no matter what I do. I'll reverse it.